MaterialDesignThemes.Wpf/SnackbarMessageQueue.cs

```diff
@@ -68,10 +57,10 @@ private async void UiElementOnMouseLeave(object sender, MouseEventArgs mouseEven
                     }
                     catch (ObjectDisposedException)
                     {
-                        /* we are we suppresing this? 
+                        /* we are we suppressing this? 
                          * as we have switched out wait onto another thread, so we don't block the UI thread, the
                          * _cleanUp/Dispose() action might also happen, and the _disposedWaitHandle might get disposed
-                         * just before we WaitOne. We wond add a lock in the _cleanUp because it might block for 2 seconds.
+                         * just before we WaitOne. We want add a lock in the _cleanUp because it might block for 2 seconds.
```

Comment:
I think this was supposed to say won't :D

Reply:
sure, you are right. I did not really read the comments. just picked the first spell-check suggestion :-)